server/www/templates/posts/post-editor.html

```diff
@@ -297,4 +297,13 @@ <h1 translate translate-values="{ type: activeForm.name, title: post.title }" ng
             </p>
         </div>
     </div>
+
+    <modal visible="userSavedPost">
+        <h1 translate>post.save_dlg_title</h1>
+        <div translate translate-values="{ 'post': post.title }">post.you_have_published</div>
+        <div class="modal-submit">
+            <button type="button" class="button-primary" ng-click="navigateToPost()" translate>post.see_post</button>
```

Comment:
If you need a button to change URL, just switch it to a link `<a href="/posts/id" class="button button-primary"></a>